drift_dev/lib/src/writer/tables/data_class_writer.dart

```diff
@@ -114,8 +114,14 @@ class DataClassWriter {
     _writeFromJson();
     _writeToJson();
 
-    // And a convenience method to copy data from this class.
+    // And convenience methods to copy data from this class.
     _writeCopyWith();
+    if (isInsertable &&
+        scope.options.dataClassToCompanions &&
```

Comment:
Do we need this option? We're not referencing the "data class to companion" function here, right? So I think it should work without this guard too.

Reply:
Good catch, thanks. I'd copied that from the `_writeToCompanion` call site, but indeed it seems to make more sense there than here.

Took that out and none of the generated code changed, and the tests I ran locally still pass. So pushed that change to the PR and we can see what CI says.